jwst/datamodels/schemas/abvega_offset.schema.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/abvegamag_offset.schema"
+title: AB to Vega magnitude offset reference file model
+allOf:
+- $ref: referencefile.schema
+- type: object
+  properties:
+    abvega_offset:
+      title: Offsets to convert AB to Vega magnitudes
+      type: object
+      properties:
+        abvega_offset:
+          title: AB_mag - Vega_mag
+          type: number
+        anyOf:
+          detector:
+            title: Detector name
+            type: string
+          filter:
+            title: Filter wheel element name
+            type: string
+          pupil:
+            title: Pupil wheel element name
+            type: string
```

Comment:
schema should have a final line with `...`

Reply:
None of the schemas in `jwst/datamodels/schemas` end with `...`.  I think you mean ASDF file must end with `...`?

Reply:
Actually both do, as it's a YAML standard.  We don't actually check for it in our checker right now, so ignore that.  We clearly have some cleaning up to do in the rest of our schemas.  =)

Reply:
I'm happy to be a trailblazer and add the trailing `...`. 😄

Reply:
I've added the trailing `...`.